### PR TITLE
Disable travis caches to solve bower problem

### PR DESCRIPTION
**Pull request by slomo**

### CODE REVIEW

**Comment by slomo** [comment]
Please not review now I am only testing the travis setup.

**Comment by xi** [comment]
Yeay, it's green! Should we merge it now?

**Comment by slomo** [comment]
Well we should do the counter test and add the caches again.

**Comment by xi** [comment]
It now seems to work again also with the cache: #2069 (or am I missing something)

File: .travis.yml [hunk 1]
```diff
@@ -14,10 +14,6 @@ notifications:
   email: false
 after_success:
   - "bin/coveralls"
-cache:
-  directories:
-    - eggs
-    - node_modules
```
Comment (joka) [inline .travis.yml]: why did you remove the python eggs directory?

Comment (slomo) [inline .travis.yml]: I did remove all caches. We suspected and issue related to caching an this pull-request is for testing travis.

**Comment by xi** [comment]
Travis seems to work fine again, even without this. closing.
